test/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+SNS_REFM_MOUNTED = os.path.isdir("/SNS/REF_M/")
+ONE_SECOND = 1000  # ms, a wait time for qtbot
```

Comment:
is this used or can it be removed?

Reply:
the purpose is to have it when we have a test containing `qtbot.wait(some_time)`, for instance:
```python
from tests.import ONE_SECOND
MainWindow().show()
qtbot.wait(2 * ONE_SECOND)  # stop execution on the main thread for two seconds
```